requirements.txt

```diff
@@ -6,7 +6,6 @@ sympy
 pytest==6.2.4
 pytest-parallel
```

Comment:
Can't comment on binary files, so commenting here =(. Why is the linux/kudu binary moved from `./warp/bin` to `./bin`? I think setup.py expects it to be in the former directory?